.pipelines/pipeline.yaml

```diff
@@ -165,16 +165,11 @@ stages:
               arch: amd64
               name: cni-dropgz-test
               os: linux
-            cni_plugin_windows2022_amd64:
-              os: windows
-              name: cni-plugin
+            cni_dropgz_test_windows2022_amd64:
               arch: amd64
-              os_version: ltsc2022
-            cni_plugin_windows2019_amd64:
+              name: cni-dropgz-test
               os: windows
-              name: cni-plugin
-              arch: amd64
-              os_version: ltsc2019
+              os_version: ltsc2022
```

Comment:
do we still support/should we still test 2019?

Reply:
We don't plan to test 2019 in PR pipeline. But we will be testing 2019 in a separate pipeline( CNI test pipeline).